vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/contextmenu/GridContextMenu.java

```diff
@@ -223,7 +224,7 @@ public Registration addGridContextMenuOpenedListener(
      * @return the callback function that is executed before opening the context
      *         menu, or {@code null} if not specified.
      */
-    public SerializablePredicate<T> getDynamicContentHandler() {
+    public SerializableBiPredicate<T, Grid.Column<T>> getDynamicContentHandler() {
```

Comment:
This would be a breaking change. Probably falls under the definition of a minor one, so I would agree with putting it into a minor, but I feel we should document it somehow.